### ISSUE TITLE
I notice you've got automated builds for CI but no nightly builds available for download?

### ISSUE THREAD

**Issue by andybak** [alicevision/meshroom#1074]
Any reason for this?

The latest release is very old and every time I try and build binaries it usually wastes half a day and fills my small SSD with libraries and build artifacts.

**Comment by andybak** [comment]
Just spotted you can download the build artifacts from appveyor. Is it worth linking to this or mentioning it in the installation docs?

**Comment by natowi** [comment]
No, at the moment this is not intended for the general public*. Most people want to test the not yet released features, but the most notable ones are in the GUI. This requires the updated qtAliceVision package. We have AliceVision and Meshroom builds as Appveyor artifacts**, but a few other required libraries are not yet available as auto-build.

*https://github.com/alicevision/meshroom/issues/1071#issuecomment-697512378
**https://github.com/alicevision/AliceVision/wiki/Windows-Snapshot-Builds

You are welcome to contribute the Appveyor build scripts for qtAliceVision: https://github.com/alicevision/QtAliceVision (to do), https://github.com/alicevision/QtOIIO/pull/15 and https://github.com/alicevision/qmlAlembic/pull/14 (waiting for merge)

**Comment by fabiencastan** [comment]
And we also need to move from appveyor to github actions on all repositories for the windows CI.

**Comment by fabiencastan** [comment]
@andybak I would love to have that in place, but we don't have the resources to do it for now.
